src/Kunstmaan/MediaBundle/Entity/CroppableMediaLink.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Kunstmaan\MediaBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Kunstmaan\AdminBundle\Entity\AbstractEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="kuma_croppable_media_link")
+ * @ORM\HasLifecycleCallbacks
+ */
+class CroppableMediaLink extends AbstractEntity
+{
+    /**
+     * @ORM\ManyToOne(targetEntity="Kunstmaan\MediaBundle\Entity\Media", cascade={"persist"})
+     * @ORM\JoinColumn(name="media_id", referencedColumnName="id")
+     * @Assert\NotNull()
+     */
+    private $media;
+
+    /**
+     * @ORM\Column(name="runtime_config", type="text", nullable=true)
+     */
+    private $runTimeConfig;
+
+    public function getMedia()
```

Comment:
As this is a new entity/class I would add typehints to all methods/parameters/etc